<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridDialog</class>
 <widget class="QDialog" name="GridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Use Grid</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>&amp;X:</string>
        </property>
        <property name="buddy">
         <cstring>xLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="xLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>&amp;Y:</string>
        </property>
        <property name="buddy">
         <cstring>yLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="yLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Use Snap</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="D:/python projects/pyside2 practice/GridDialog.qrc">
         <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="D:/python projects/pyside2 practice/GridDialog.qrc">
         <normaloff>:/images/cancel.jpg</normaloff>:/images/cancel.jpg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="D:/python projects/pyside2 practice/GridDialog.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GridDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GridDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
